src/vm/ClrEtwAll.man

```diff
@@ -3307,9 +3383,26 @@
                            task="CodeSymbols"
                            symbol="CodeSymbols" message="$(string.RuntimePublisher.CodeSymbolsEventMessage)"/>
 
-                   <event value="270" version="0" level="win:Informational"  template="EventSource"
+                    <event value="270" version="0" level="win:Informational"  template="EventSource"
                            keywords="EventSourceKeyword"
                            symbol="EventSource" />
+
+                    <!-- Tiered compilation events 280-289 -->
+                    <event value="280" version="0" level="win:Informational" template="TieredCompilationSettings"
+                           keywords="TieredCompilationKeyword" task="TieredCompilation" opcode="Settings"
+                           symbol="TieredCompilationSettings" message="$(string.RuntimePublisher.TieredCompilationSettingsEventMessage)"/>
+                    <event value="281" version="0" level="win:Informational" template="TieredCompilationEmpty"
```

Comment:
Is the template name "TieredCompilationEmpty" intentional?  It does not match Task+Opcode which would be "TieredCompilationPause".

Reply:
Yes it's similar to `EmptyTraceData`, it could be reused in the future for TC events that don't have any data. If data specific to `Pause` needs to be added, it would be renamed (if not reused) or copied to be specific to the `Pause` event.